src/experiments.js

```diff
@@ -595,6 +595,9 @@ class PersonalizationBlock extends HTMLElement {
 				},
 			}, false );
 		} );
+
+		// Dispatch the altisBlockContentChanged event.
+		window.dispatchEvent( new Event( 'altisBlockContentChanged' ) );
```

Comment:
I think this is needed for the AB test block too?

Reply:
Will have a look, this is currently in place for when previewing the page and toggling the tabs.

Reply:
Still need to move this one :)

Reply:
Sneaky! Got it moved accordingly now.